CHANGELOG.md

```diff
@@ -43,6 +45,10 @@ avifEncoder struct.
 * avifImageRGBToYUV() and avifImageYUVToRGB() handle avifImage bit depths 8, 10,
   12 and now also 16. Files read by apps/shared/ can output 16-bit avifImage
   instances.
+* avifImageCreate(), avifImageCreateEmpty(), avifEncoderCreate() and other
+  internal functions now return NULL in case of AVIF_RESULT_OUT_OF_MEMORY.
```

Comment:
Since these functions don't report an error code, it is a little confusing to mention the error code `AVIF_RESULT_OUT_OF_MEMORY` here. I suggest we say "in case of out-of-memory error" or "in case of memory allocation error".